week-1/implement/max.js

```diff
@@ -0,0 +1,6 @@
+
+function getMax(array){
+    return Math.max.apply(null, array);
```

Comment:
Nicely done. Can you think of another way of getting the max number from an array? without searching on Google. for example, using the reduce method or normal for loop?
Is good exercise to try different ways to do things, Now that you have learned more and progressed well as I can see!

Reply:
the for loop would work, looping through each number and checking if its the largest or not and returning that then.